charts/ckan/values.yaml

```diff
@@ -0,0 +1,120 @@
+global:
+  imageRegistry: ""
+  imagePullSecrets: []
+  storageClass: ""
+
+nameOverride: ""
+fullnameOverride: ""
+
+defaultPodSecurityContext: &defaultPodSecurityContext
+  fsGroup: 1000
+  runAsUser: 1000
+  runAsGroup: 1000
+  runAsNonRoot: true
+  fsGroupChangePolicy: OnRootMismatch
+
+defaultSecurityContext: &defaultSecurityContext
+  runAsNonRoot: true
+  readOnlyRootFilesystem: true
+  allowPrivilegeEscalation: false
+  privileged: false
+  capabilities:
+    drop:
+      - ALL
+  runAsGroup: 1000
+  runAsUser: 1000
+
+ckan:
+  replicas: 1
```

Comment:
Defaults should be production-ready, so this should probably be at least 2?

Reply:
Unfortunately, this is currently not possible there as there is also the problem with the RWO volume.